insights/parsers/swift_conf.py

```diff
@@ -0,0 +1,50 @@
+"""
+swift_conf - files under /etc/swift/
+====================================
+
+This module provides parsers for swift config files under /etc/swift directory.
+"""
+
+from .. import IniConfigFile, parser
+
+
+@parser("proxy_server.conf")
+class SwiftProxyServerConf(IniConfigFile):
+    """
+    This class is to parse the content of the ``/ etc / swift / proxy - server.conf``.
+    The swift proxy - server configuration file ``/ etc / swift / proxy - server.conf`` is in the standard 'ini'
+    format and is read by the IniConfigFile parser. Sample configuration file::
+
+        [DEFAULT]
+        bind_port = 8080
+        bind_ip = 172.20
+        .15
+        .20
+        workers = 0
+        [pipeline:main]
+        pipeline = catch_errors
+        healthcheck
+        proxy - logging
+        cache
+        ratelimit
+
+        [app:proxy - server]
+        use = egg:swift  # proxy
+        set
+        log_name = proxy - server
+        set
+        log_facility = LOG_LOCAL1
+
+        [filter:catch_errors]
+        use = egg:swift  # catch_errors
+
+    Examples:
+    >> > proxy_server_conf = shared[SwiftProxyServerConf]
```

Comment:
One more indent is needed for the lines after the `Examples:` and better remove the space in the `>> >`.